doc/getting-started/run.md

```diff
@@ -5,16 +5,7 @@
 Castor provides a `run()` function to execute external processes.
 
 ```php
-use Castor\Attribute\AsTask;
-
-use function Castor\run;
-
-#[AsTask()]
-function foo(): void
-{
-    run('my-script.sh');
-    run(['php', 'vendor/bin/phpunit', '--filter', 'MyTest']);
-}
+{% include "/examples/basic/run/run.php" start="<?php\n\nnamespace run;\n\n" %}
```

Comment:
I think we could make an example with real command, so this example would be tested